R/initModel.R

```diff
@@ -80,9 +89,9 @@ initModel <- function(config = NULL,
                       readDirect = TRUE)
     title <- cfg[["title"]]
   } else {
-    if (!is.null(restart)) {
-      message("Restart options were given, but no existing path was specified. Starting a new run.")
-      restart <- NULL
+    # Start a new run
```

Comment:
In case a user mis defines `restart`. Alternatively, we could throw an error.
```suggestion
    # Start a new run
    restart <- FALSE
```

Reply:
Good point, thanks!